src/main/java/DayFour.java

```diff
@@ -5,21 +5,25 @@
 public class DayFour {
     public String findInputs(Integer total, String listOfCodes) {
         List<String> codesList = Arrays.asList(listOfCodes.split(","));
+//indentation and typing
 String toCheck;
         String result = "";
         ArrayList<String> replacementNumbers = new ArrayList<>();
         DayThree dayThree = new DayThree();
+        //why 12?
         for (int noun = 12; noun < 100; noun++) {
 
             for (int verb = 0; verb < 100; verb++) {
                 codesList.set(1, String.valueOf(noun));
                 codesList.set(2, String.valueOf(verb));
+                //naming - why replace? - line length
```

Comment:
I'm guessing all the replaces are to cope with the default formatting of the List.toString method? I think it would be better to have the method take an array or a list to save all of this inline parsing.